src/test/regress/expected/multi_utility_statements_2.out

```diff
@@ -0,0 +1,363 @@
+--
```

Comment:
I cannot even understand the necessity for adding this test, the diff is suspicious:

```
diff expected/multi_utility_statements.out expected/multi_utility_statements_2.out 
195a196
> ERROR:  tables declared WITH OIDS are not supported
198,206c199
<  customer_key |       c_name       |                c_address                
< --------------+--------------------+-----------------------------------------
<             3 | Customer#000000003 | MG9kdTD2WBHm
<            14 | Customer#000000014 | KXkletMlL2JQEA 
<            30 | Customer#000000030 | nJDsELGAavU63Jl0c5NKsKfL8rIJQQkQnYL2QJY
<            59 | Customer#000000059 | zLOCP0wh92OtBihgspOGl4
<           106 | Customer#000000106 | xGCOEAUjUNG
< (5 rows)
< 
---
> ERROR:  current transaction is aborted, commands ignored until end of transaction block
```

So, how could pg12 error out with ` tables declared WITH OIDS are not supported`? 

Could this be an artifact of some intermediate works you've done?